### PR TITLE
Entrypoint for talisker gevent workers

### PR DESCRIPTION
**Pull request by nottrobin**
Fixes https://github.com/ubuntudesign/base-squad/issues/106

QA
--

``` bash
./run
```

See that it mentions "Booting worker" 3 times. Go to http://localhost:8203 - browse around, check the site works. Then:

``` bash
docker build --tag juju-docs --build-arg COMMIT_ID=`git rev-parse HEAD` .
docker run -ti -p 8099:80 juju-docs
```

See that it mentions "Booting worker" 3 times. Go to http://localhost:8099 and check the site works.

Also, check the `X-VCS-Revision` shows correctly:

``` bash
$ curl -Is localhost:8099 | grep VCS
X-VCS-Revision: {commit_hash}
```

### CODE REVIEW

**Comment by pmatulis** [comment]
@nottrobin ELI5, what is this PR for?

**Comment by nottrobin** [comment]
@pmatulis what's ELI5? This is the way we have been asked to run all python apps in Kubernetes, by IS. It's so the application can handle more requests by running multiple processes.

**Comment by pmatulis** [comment]
@nottrobin Thanks.

ELI5: Explain like I'm five.
